okio/src/commonMain/kotlin/okio/FileHandle.kt

```diff
@@ -171,6 +171,37 @@ abstract class FileHandle(
     return source.position - bufferSize
   }
 
+  /**
+   * Change the position of [source] in the file to [position]. The argument [source] must be either
+   * a source produced by this file handle, or a [BufferedSource] that directly wraps such a source.
+   * If the parameter is a [BufferedSource], it will skip or clear buffered bytes.
+   */
+  @Throws(IOException::class)
+  fun position(source: Source, position: Long) {
```

Comment:
should this be reposition? I’m tempted by that because it’s symmetric with size/resize

Reply:
done.